cdap-data-fabric/src/main/java/co/cask/cdap/data2/sql/PostgresSqlStructuredTableAdmin.java

```diff
@@ -46,15 +51,24 @@ public PostgresSqlStructuredTableAdmin(DataSource dataSource) {
 
   @Override
   public void create(StructuredTableSpecification spec) throws IOException, TableAlreadyExistsException {
+    LOG.info("Creating table {}", spec);
```

Comment:
I think you should only log this if you actually create it, that is, if it does not already exist,

Reply:
+1